feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/manualBackup/warning/ManualBackupWarningViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package io.novafoundation.nova.feature_account_impl.presentation.manualBackup.warning
+
+import androidx.lifecycle.viewModelScope
+import io.novafoundation.nova.common.base.BaseViewModel
+import io.novafoundation.nova.common.mixin.condition.ConditionMixinFactory
+import io.novafoundation.nova.feature_account_impl.R
+import io.novafoundation.nova.feature_account_impl.presentation.AccountRouter
+import io.novafoundation.nova.feature_account_impl.presentation.manualBackup.common.ManualBackupAccountToBackupPayload
+
+const val CONDITION_ID_1 = 0
+const val CONDITION_ID_2 = 1
+const val CONDITION_ID_3 = 2
```

Comment:
This seems like a part that will be duplicated each time we use `ConditionsMixin`
My suggestion is to have an extension method

`ConditionsMixin.setupConditions(varag conditionInputs: Checkable)` for usage in the ui

Reply:
Nice solution!

But it's not enough to implement Checkable interface since it doesn't have observe method. Added `CheckableListener `for this case